lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -12,6 +12,7 @@
 
 #include "circt/Dialect/FIRRTL/FIRRTLOps.h"
 #include "circt/Dialect/FIRRTL/FIRRTLAnnotations.h"
+#include "circt/Dialect/FIRRTL/FIRRTLAttributes.h"
```

Comment:
Why is this necessary? (I would've thought to see some change to this file...)

Reply:
This is because `FIRRTL.cpp.inc` are using the new attribute we introduced :eyes:

Reply:
Got it. That makes sense and this is then the minimal location where it should be put. :100: